src/jsonpatcherproxy.js

```diff
@@ -177,35 +168,23 @@ const JSONPatcherProxy = (function() {
             it is a good idea to remove this flag if we come across it here, in deleteProperty trap.
```

Comment:
The `deleteProperty` in the comment is outdated now. Should be `trapForDeleteProperty`.